index.html

```diff
@@ -370,6 +370,27 @@ <h4>Multikey</h4>
   ]
 }
           </pre>
+
+          <p>
+            The `secretKeyMultibase` property represents a Multibase-encoded Multikey
+            expression of a P-256 or P-384 secret key (also sometimes referred to
+            as a private key). The encoding of a P-256 secret key is
+            the two-byte prefix `0x8626` (the varint expression of `0x1306`) followed
+            by the 32-byte secret key data.
+            The 35-byte value is then encoded using base58-btc (`z`) as the prefix.
```

Comment:
```suggestion
            The 34-byte value is then encoded using base58-btc (`z`) as the prefix.
```

Reply:
Hmm, the secret key prefix looks wrong from my code [ECDSAKeyCheck.js](https://github.com/Wind4Greg/ECDSA-SD-TestVectors/blob/main/ECDSAKeyCheck.js) I had

```javascript
const P256_PUB_PREFIX = 0x1200;
const P384_PUB_PREFIX = 0x1201;
const P256_PRIV_PREFIX = 0x1306;
const P384_PRIV_PREFIX = 0x1307;
```

And leading bytes and lengths produce by program output were:

```text
Multicodec leading bytes in hex for P-256 and P-384 compressed public keys:
Public P-256 leading bytes: 8024
Public P-384 leading bytes: 8124

Multicodec leading bytes in hex for P-256 and P-384 private keys:
Private P-256 leading bytes: 8626
Private P-384 leading bytes: 8726

P256 key example:
P-256 private key length: 32
P-256 private key hex:
c9afa9d845ba75166b5c215767b1d6934e50c3db36e89b127b8a622b120f6721
Private P-256 encoded multikey:
z42twTcNeSYcnqg1FLuSFs2bsGH3ZqbRHFmvS9XMsYhjxvHN

P-256 Pubic key length 33
P-256 public key in hex:
0360fed4ba255a9d31c961eb74c6356d68c049b8923b61fa6ce669622e60f29fb6
Public P-256 encoded multikey:
zDnaepBuvsQ8cpsWrVKw8fbpGpvPeNSjVPTWoq6cRqaYzBKVP

P-384 private key length: 48
6b9d3dad2e1b8c1c05b19875b6659f4de23c3b667bf297ba9aa47740787137d896d5724e4c70a825f872c9ea60d2edf5
Private P-384 encoded multikey:
z2fanyY7zgwNpZGxX5fXXibvScNaUWNprHU9dKx7qpVj7mws9J8LLt4mDB5TyH2GLHWkUc

P-384 Pubic key length 49
P-384 public key in hex:
02ec3a4e415b4e19a4568618029f427fa5da9a8bc4ae92e02e06aae5286b300c64def8f0ea9055866064a254515480bc13
P-384 encoded multikey:
z82LkuBieyGShVBhvtE2zoiD6Kma4tJGFtkAhxR5pfkp5QPw4LutoYWhvQCnGjdVn14kujQ
```